lib/Optimizer/Partition.cpp

```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Optimizer/Partition.h"
+
+#include "glow/Graph/Graph.h"
+#include "glow/Graph/Utils.h"
+
+#include "llvm/ADT/DenseMap.h"
+#include "llvm/Support/Casting.h"
+
+using namespace glow;
```

Comment:
Add `using llvm::isa;` and replace `llvm::isa` by `isa` in the code. This is what we usually do in the Glow codebase.